app/views/shared/_topnav.html.erb

```diff
@@ -1,16 +1,16 @@
 <% if Settings.enable_logins %>
   <ul class="nav nav-tabs mb-3">
     <li class="nav-item">
-      <a class="nav-link <%= current_controller?("passwords") ? 'active' : '' %>" aria-current="page" href="<%= new_password_path %>"><%= _('Passwords') %></a>
+      <a class="nav-link <%= @text_tab ? 'active' : '' %>" aria-current="page" href="<%= new_push_path(tab: 'text') %>"><%= _('Passwords') %></a>
     </li>
     <% if Settings.enable_file_pushes %>
       <li class="nav-item">
-        <a class="nav-link <%= current_controller?("file_pushes") ? 'active' : '' %>" href="<%= new_file_push_path %>"><%= _('Files') %></a>
+        <a class="nav-link <%= @files_tab ? 'active' : '' %>" href="<%= new_push_path(tab: 'files') %>"><%= _('Files') %></a>
```

Comment:
I prefer to use `file` as tab name. But, this tab is named as `files` in the Premium one. So, I have used same name. Do you want me to change it as `file`? @pglombardo

Reply:
Let's keep it as "Files" for now and we can change later if we decide.  I think of the tab as "working with Files" as in you can push multiple files.